src/adapters/CKEditorAdapter.ts

```diff
@@ -20,22 +20,45 @@ import {HasEditorID, ContentExtractionResult} from "./AdapterInterface";
 
 export class CKEditorAdapter extends AbstractRichtextEditorAdapter {
   editorId: string;
+  editorVersion = 4;
+  editorIdHashed: string;
 
   constructor(conf: HasEditorID) {
     super(conf);
     this.editorId = conf.editorId;
+    if(typeof window["CKEDITOR"] == 'undefined'){
+      this.editorVersion = 5;
+    }
+    this.editorIdHashed = "#" + this.editorId;
   }
 
   getEditor(): CKEDITOR.editor {
-    const ckeditor = CKEDITOR.instances[this.editorId as any];
-    if (!ckeditor) {
-      throw new Error(`Can't find ckeditor with id '${this.editorId}'`);
+    if(this.editorVersion === 4){
+      const ckeditor = CKEDITOR.instances[this.editorId as any];
+      if (!ckeditor) {
+        throw new Error(`Can't find ckeditor with id '${this.editorId}'`);
+      }
+      return ckeditor;
+    }
+    // CKEditor Version 5 detected
+    let editorDomElement = document.querySelector(this.editorIdHashed)!;
+    const isInlineEditor = editorDomElement.classList.contains('ck-editor__editable');
+    if(!isInlineEditor){
+      // Classic editor detected
+      editorDomElement = editorDomElement.nextElementSibling!.querySelector('.ck-editor__editable')!;
```

Comment:
Is the usage of `nextElementSibling` observation based or does it have a reason?

Reply:
It does have a reason. If a classic editor is in use, the next element is the editor.